app.js

```diff
@@ -7,8 +7,17 @@ const bodyParser = require('body-parser')
 const exphbs = require('express-handlebars')
 const videos = require('./models/videosDb')
 
+const sassMiddleware = require('node-sass-middleware')
+
 const app = express()
 
+app.use(sassMiddleware({
+    src: path.join(`${__dirname}/view/sass`),
+    dest: path.join(`${__dirname}/view/css`),
+}))
+
+app.use(express.static('view/css'))
```

Comment:
Me parece que o diretório de `statics` já está configurado no projeto ([aqui](https://github.com/aceleradora-TW/conhecimento-livre/blob/master/app.js#L23), mais precisamente).

Me parece fazer sentido que a pasta destino seja `public/css`  ao invés de `view/css`, já que a pasta `public` já está exposta na configuração anterior à esta PR. Dessa forma, se não estou enganado, nos layouts html/handlebars das views, vocês podem incluir os arquivos de CSS de vocês com um caminho "absoluto", da mesma forma que vocês já fazem (e que funciona, pois os arquivos css são gerados na pasta views, onde ficam os templates):

``` html
<head>
    <link rel="stylesheet" href="/estilo-afude-prafrentex-2017.css"></link>
</head>
```

O que lhes parece?

Reply:
THIS!

Reply:
ok